### ISSUE TITLE
Reference investigation: Together (Vocal Remix) / Someday / Day That Never Came

### ISSUE THREAD

**Issue by Niklink** [hsmusic/hsmusic-data#452]
> Westminster Chimes in Together: Vocal Remix, sounds like
https://twitter.com/barkbarkclark/status/1544760860534460416
my heart is still in this, listen even if you don't know what psycholonials is. only have time for one? listen to the bonus track at the end
https://twitter.com/barkbarkclark/status/1544761280203874304
it's the second time in my career i snuck "westminster chimes" into a song just because my blood seethes with evil and i love clocks
Also bits of commentary:
https://twitter.com/Luluco_G/status/1544769603632521224
I guess that makes sense since lots of tracks were made before the game, right? I 100% agree with this!
I've actually listened to this a lot outside of the game!!
https://twitter.com/barkbarkclark/status/1544778406054621184
yeah! we felt like synthesis and recontextualization of pieces would be the right direction for the game's narrative

To summarize:
Day That Never Came is the original
Someday adds 30ish seconds of new material to the end
Together (used in Psycholonials) is shortened to loop right before the lyrics kick in
Together (Vocal Remix) (also used in Psycholonials) is a shortened, loopable version of the original
I cannot find a single goddamn westminster chime anywhere in any of these
I thought to try Ephemeral Muse (Final Remix), the last publicly viewable track in the album, and although it felt a lot more promising ended nowhere as well
please help

**Comment by towerofnix** [comment]
Detailed investigation from Lan, which sheds no light on Westminster Chimes: [#music-discussion](https://discord.com/channels/749042497610842152/749067042250031214/1221468080002175086)
